### DISCUSSION TITLE
http proxy

### DISCUSSION THREAD

**Discussion by xenon-ua** [uNetworking/uWebSockets.js#882] [Q&A] [answered]
Hello. 
How can I proxy the received request to another server? I tried to find an example in the code or a solution to this question in the discussions, but the search unfortunately did not return any result.
I would be grateful for help.

**Comment by e3dio** [comment]
https://developer.mozilla.org/en-US/docs/Web/API/fetch

**Comment by jordan-breton** [comment, marked as answer]
I released a reverse proxy using uWebSockets.js, having some issues with a client of mine that had a codebase tighlty coupled with an express application, but needed to get bettter performances on his websocket server since his app started to use them intensively.

https://github.com/jordan-breton/uws-reverse-proxy

Feels free to report any issue/suggestion.

**Reply by xenon-ua** [reply]
Hello. It works. But some requests are aborted when I use autocannon. I am getting an error  `Gateway abortion`

**Reply by jordan-breton** [reply]
This error occurs when the recipient server abort a request before returning a proper response to the proxy, or when the request is aborted abruptly without sending the full response. It looks like your app reached its maximum capacity and started to struggle to send responses back to the proxy or just to accept new requests.

At first glance there is nothing that you should be worried about unless this error happens with a low req/s rate. 

You still should create an issue on [the repository](https://github.com/jordan-breton/uws-reverse-proxy) with a minimal reproducible example.
